### DISCUSSION TITLE
How to put username.github.io site into maintenance mode

### DISCUSSION THREAD

**Discussion by chaseterzian** [community/community#57877] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, my username.github.io site is very out of date and I would like to temporarily take it down so the public can not access it. How do I do this?

**Comment by radar-blip** [comment]
You can open the `Repository`, click on `Pages`, click the `3-dotted menu` and `Unpublish site` (like below):

![Screenshot from 2023-06-12 17-28-31](https://github.com/community/community/assets/39196767/d26af25f-a1e8-4888-91ef-2a481955b2de)
